code/game/objects/items/tools/synth_bracer/abilities/framework.dm

```diff
@@ -0,0 +1,182 @@
+/* -- ACTIVATABLE ACTIONS -- */
+
+/datum/action/human_action/activable/synth_bracer
+	icon_file = 'icons/obj/items/synth/bracer.dmi'
+	var/mob/living/carbon/human/synth
+	var/obj/item/clothing/gloves/synth/synth_bracer
+	var/charge_cost = 0
+	var/handles_cooldown = FALSE
+	var/handles_charge_cost = FALSE
+	/// What cateogry of action it is. Can only have one active action from each type.
+	var/category = SIMI_SECONDARY_ACTION
+	var/human_adaptable = FALSE
+
+/datum/action/human_action/activable/synth_bracer/give_to(user)
+	/// never add a check to see if the synth has gloves on, because they shouldn't have these abilities while not wearing gloves. it should runtime to let us know
+	synth = user
+	synth_bracer = synth.gloves
```

Comment:
I don't understand the comment - do you mean intent is to have it runtime in other procs if the gloves are not valid ?

Reply:
This is a comment left from geeves so I have no idea either